compose-stability-analyzer-idea/src/main/kotlin/com/skydoves/compose/stability/idea/toolwindow/ComposableStabilityCollector.kt

```
@@ -0,0 +1,178 @@
+                .getDocument(ktFile)
+              document?.getLineNumber(function.textOffset)?.plus(1) ?: 0
+            } catch (e: Exception) {
+              0
+            }
+            return Triple(virtualFile.path, virtualFile.name, line)
+          }
+        }
+      }
+
+      return Triple("Unknown", "Unknown.kt", 0)
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)